clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/isolated/IsolatedIngestionServer.java

```diff
@@ -436,21 +436,21 @@ public synchronized LeaderFollowerPartitionStateModel.LeaderSessionIdChecker get
   }
 
   // Set the topic partition state to be unsubscribed.
-  public void setPartitionToBeUnsubscribed(String topicName, int partition) {
+  public void setResourceToBeUnsubscribed(String topicName, int partition) {
     topicPartitionSubscriptionMap.putIfAbsent(topicName, new VeniceConcurrentHashMap<>());
     topicPartitionSubscriptionMap.get(topicName).putIfAbsent(partition, new AtomicBoolean(false));
     topicPartitionSubscriptionMap.get(topicName).get(partition).set(false);
   }
 
   // Set the topic partition state to be subscribed.
-  public void setPartitionToBeSubscribed(String topicName, int partition) {
+  public void setResourceToBeSubscribed(String topicName, int partition) {
```

Comment:
Nitpick: the code in this function is pretty verbose... can we consider doing it more succinctly, e.g.:

```
    topicPartitionSubscriptionMap.computeIfAbsent(topicName, s -> new VeniceConcurrentHashMap<>())
        .computeIfAbsent(partition, p -> new AtomicBoolean(true))
        .set(true);
```

Same comment for `setResourceToBeUnsubscribed`